apps/ccd/automation/kustomization.yaml

```diff
@@ -1,6 +1,9 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
+  - ../ccd-admin-web/image-repo.yaml
+  - ../ccd-admin-web/image-policy.yaml
```

Comment:
Hey, these two have just been duplicated and need to be removed

Reply:
done